client/hijack.go

```diff
@@ -66,7 +66,8 @@ func (cli *Client) setupHijackConn(req *http.Request, proto string) (_ net.Conn,
 	}
 
 	ctx, span := tp.Tracer("").Start(ctx, req.Method+" "+req.URL.Path)
-	span.SetAttributes(semconv.HTTPClientAttributesFromHTTPRequest(req)...)
+	// TODO(thaJeztah): should this also set trace.SpanKindClient like containerd's tracing package does (or use containerd's tracing.WithHTTPRequest?; https://github.com/containerd/containerd/blob/8c087663b0233f6e6e2f4515cee61d49f14746a8/tracing/tracing.go#L38-L47)
```

Comment:
I stumbled upon that containerd code when looking how the other projects used this code (https://github.com/containerd/containerd/commit/5082fb3958ae81ce02102031b6c4376041d1b9fb), and noticed the code in containerd also set that property; https://github.com/moby/moby/blob/80a9fc6d36430b067babe841e8332888e24e590b/vendor/github.com/containerd/containerd/tracing/tracing.go#L38-L47


That code was originally added as part of https://github.com/containerd/containerd/commit/3b87d46ce291565012e30956cb3772026c39d58d#diff-d6dd575a9d10716db9ac385bf5f394a2e65d638311159e8726b3200fcc596989R572-R577, but perhaps someone more familiar with this knows if we should add the same


/cc @cpuguy83 @tonistiigi

Reply:
Yes this should set the span kind.